alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -2967,6 +2967,49 @@ public static function addDeliveryTagToEntry(entry $entry)
 			$entry->save();
 		}
 	}
+
+	public static function handleDelayedNotification($object)
+	{
+		$delayedLockedJobs = [];
+		switch (true)
+		{
+			case ($object instanceof entry):
+			{
+				$delayedLockedJobs = BatchJobLockPeer::retrieveByEntryIdAndStatus($object->getEntryId(), BatchJob::BATCHJOB_STATUS_DELAYED);
```

Comment:
let's change the naming to delayedJobLocks so it is more clear that the objects are locks and not the jobs themselves